README.md

```diff
@@ -1,102 +1,114 @@
+
+    ```
     $ git clone git@github.com:caktus/hosted-rapidpro
     $ git clone https://github.com/rapidpro/rapidpro.git
+    ```
+
+3. Create a virtual environment and activate it.
+
+4. Local set-up work is encapsulated in a single shell script:
+
+    ```
     $ cd rapidpro
     $ sh ../hosted-rapidpro/local_rapidpro.sh
+    ```
 
-Then read the rest of this README.
+5. Edit ``temba/settings.py`` and add a few new settings:
 
-Running the server locally
---------------------------
+    ```
+    HOSTNAME = "MYSUBDOMAIN.ngrok.io"
+    TEMBA_HOST = "MYSUBDOMAIN.ngrok.io"
+    ALLOWED_HOSTS = ["MYSUBDOMAIN.ngrok.io"]
+    SEND_MESSAGES = True
+    ```
 
-To start the server:
+5. In a separate terminal, start the server:
 
-    $ cd ..../rapidpro
-    $ workon rapidpro
+    ```
     $ python manage.py runserver
+    ```
 
-Running celery locally
-----------------------
-
-To start celery:
-
-    $ cd ..../rapidpro
-    $ workon rapidpro
-    $ python manage.py celery worker --beat --loglevel=info
-
-Running locally using ngrok and twilio
---------------------------------------
-
-* Get access to a Twilio account
-* Get a new Twilio phone number.
-* Go to the account settings page
-  (https://www.twilio.com/user/account/settings)
-  and write down the AccountSID and
-  AuthToken for later use, or keep this page open
-  so you can get them later.
-* Get a *paid* ngrok account. (Cakti: use the *Google* 
-  login on the ngrok web site rather than creating a
-  new account directly.  Use your Caktus email account.)
-* Install ngrok (https://ngrok.com/). These instructions
-  were developed using ngrok 2.0.25, so try running:
-  
-      ngrok version
-  
-  and make sure your version isn't older than that.
-* Tell ngrok to use your web ngrok account: Go to the
-  auth page (https://dashboard.ngrok.com/auth), copy
-  the long random "authtoken" string, and run:
-  
-     ngrok authtoken <YOURAUTHTOKEN>
-   
-* Start ngrok in a local window, picking an 
-  arbitrary subdomain (any string of letters you
-  want, it doesn't have to match anything else):
-
-    $ ngrok http -subdomain=MYSUBDOMAIN 8000
-
-* ngrok will show you the address it's forwarding from -
-  probably http://SUBDOMAIN.ngrok.io.  Edit your local
-  settings.py and set HOSTNAME and TEMBA_HOST to
-  that hostname (not the full URL, just the hostname), e.g.:
-
-    HOSTNAME = 'SUBDOMAIN.ngrok.io'
-    TEMBA_HOST = 'SUBDOMAIN.ngrok.io'
-  
-* Also turn on sending in settings.py:
+6. In a separate terminal, start celery:
 
-    SEND_MESSAGES = True
+    ```
+    $ python manage.py worker --beat --loglevel=info
+    ```
 
-* Start rapidpro (see above). Make sure it's running on port
-  8000, or that you've changed the command you used to start
-  ngrok to connect to the port where rapidpro is running.
+7. In a separate terminal, start ngrok on the same port as your development
+   server:
 
-* Start celery (see above)
+    ```
+    $ ngrok http -subdomain MYSUBDOMAIN 8000
+    ```
```

Comment:
This might be a good place to add `If you get an error like xxxxxxxxxxxxxxxxx, ngrok is trying to use your free account instead of the paid Caktus account; go back to step N in the Ngrok Account section of this document to fix it.